### ISSUE TITLE
[Fluid] Add CFL output

### ISSUE THREAD

**Issue by rubenzorrilla** [KratosMultiphysics/GiDInterface#679]
This issue is a reminder to add the new CFL output capability in a similar manner we did for the drag one.

The plan is to do this after the upcoming release.

**Comment by jginternational** [comment]
Have you got a definition of the desired gui and the desired json?

**Comment by rubenzorrilla** [comment]
> Have you got a definition of the desired gui and the desired json?

Yes. We've just merged it. I will show you after the release :wink: .

**Comment by jginternational** [comment]
ping

**Comment by mpentek** [comment]
The CFL is meanwhile dependent on the StatisticsApplication, so I am not sure if adding it is the best option or how to do it properly. It needs that App compiled and, in case of restart, the MainKratos needs to import it additionally.

**Comment by rubenzorrilla** [comment]
About having the app, I think we could pack it in the GiD binaries. Is is not a large application and comes with some features that could be useful for postprocessing.

Concerning the import, we could modify the `MainKratos.py` to import the application if available. This would keep the current behavior (current GUI users with custom compilations wouldn't realize) but open the possibility of restarting statistics-dependent cases run with the GUI binaries.
